### PR TITLE
Install docs

### PR DESCRIPTION
**Pull request by LeaRieck**

### CODE REVIEW

File: DyssolInstallers/Installer_Src_x86_x64/wsl_create_docs.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# create build and install directories
+if (!(Test-Path $src_path\$build_path)) {
+	New-Item -itemType Directory -Path $src_path\$build_path
+}
+if (!(Test-Path $src_path\$install_path)) {
+	New-Item -itemType Directory -Path $src_path\$install_path
+}
+
+#install dependencies
```
Comment (vasylskorych) [inline DyssolInstallers/Installer_Src_x86_x64/wsl_create_docs.ps1:25]: Remove commented code

File: DyssolInstallers/Installer_Src_x86_x64/wsl_create_docs.ps1 [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#Write-Host "Install dependencies via WSL"
+#bash -c "pip install -U sphinx sphinx-rtd-theme breathe"
+#"apt-get install doxygen graphviz &&  apt update && apt install python3-pip && 
+# compile docs
+Write-Host "Generating documentation via WSL"
+wsl -e bash -lic "cd $src_path_wsl/$build_path; cmake .. -DCMAKE_INSTALL_PREFIX=../$install_path -DBUILD_BINARIES=NO; make doc; make install"
+#bash -c "cd $src_path_wsl/$build_path && cmake .. -DCMAKE_INSTALL_PREFIX=../$install_path -DBUILD_BINARIES=NO && make doc && make install"
```
Comment (vasylskorych) [inline DyssolInstallers/Installer_Src_x86_x64/wsl_create_docs.ps1:32]: This also

**Review by vasylskorych — CHANGES_REQUESTED** [review]

File: Utilities/BuildVersion.h [hunk 3]
```diff
@@ -1,2 +1,2 @@
-#pragma once
+#pragma once
```
Comment (gladk) [inline Utilities/BuildVersion.h:1]: Please revert this line ending change.

**Review by vasylskorych — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -29,13 +29,21 @@ Run the provided installer and follow the instructions.
...
 - Compile and build external libraries: zlib, HDF5, SUNDIALS, graphviz. To do this, navigate to `Dyssol/ExternalLibraries/` and execute file `RunAll.bat`. It will start building all the required libraries by executing files `CompileZLib.ps1`, `CompileHDF5.ps1`, `CompileSundials.ps1`, `CompileGraphviz.ps1`. To use the scripts, the following requirements apply: Visual Studio 16 2019, CMake, PowerShell 5.0.
 - Open `Dyssol/Dyssol.sln` with Visual Studio and build the solution.
-- To generate installers in Solution Explorer, go to `Installers` and right click on the version you want to compile, choose Build. The built `.exe` installer locates in `Dyssol/DyssolInstallers/Installers`.
+
+#### Procedure to generate installers on Windows
+- install required build tools on WSL
+```sh
+$ sudo apt-get install doxygen graphviz
+$ pip install -U sphinx sphinx-rtd-theme breathe
```
Comment (gladk) [inline README.md:44]: If you use apt installer already, please try to use it also for sphinx theme. 

sudo apt-get install doxygen graphviz python3-breathe python3-sphinx-rtd-theme

Comment (vasylskorych) [reply README.md:44]: I would like to, but standard packages are ancient - both Ubuntu 18 and 20 have Sphinx 1.X.X, and ist output is rather ugly.

**Pull request merged**